src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -176,7 +188,7 @@ private function parseDefinition($id, $service, $file)
         static::checkDefinition($id, $service, $file);
 
         if (isset($service['alias'])) {
-            $public = !array_key_exists('public', $service) || (bool) $service['public'];
+            $public = array_key_exists('public', $service) ? (bool) $service['public'] : !empty($defaults['public']);
```

Comment:
I had services defined: (Symfony 3.1.8):
```yaml
foo:
    class: My\FooService

bar:
    alias: foo
```
I could access `bar` service in my controller because it was public by default. After this change here my aliased service is private by default, thus the service is not accessible in controller.

Shouldn't this be mentioned somewhere? I couldn't find any info about it.